<launch>
  <arg name="bot1_name" value="agent1"/>
  <arg name="spawn_model" value="false" />
  <arg name="x1" value="0.0" />
  <arg name="y1" value="5.0" />

  <param name="bot1_name" value="$(arg bot1_name)"/>
  <arg name="bot1_ip" default="turtlebot4" />

  <machine name="Agent1" user="turtlebot" address="$(arg bot1_ip)" password="turtlebot"
    env-loader="~/the_hunting_ground/src/path_following_turtle/turtle_env.sh" default="true" />

  <!--    Start Up Agent 1  -->
  <include file="$(find path_following_turtle)/launch/include/kobuki.launch.xml">
    <arg name="robot_name" value="$(arg bot1_name)" />
    <arg name="spawn_model" value="$(arg spawn_model)" />
    <arg name="robot_position" value="-x $(arg x1) -y $(arg y1)" />
    <arg name="use_machine" value="true" />
    <arg name="machine" value="Agent1" />
    <!--   If robot isn't being spawned, robot_position is optional    -->
  </include>

  <!-- Retrieve Gazebo states for agents -->
  <!-- <node name="agent1_state_pubber"
        pkg="path_following_turtle"
        type="agent1_state_server"
        respawn="true" >
  </node> -->

  <node name="bot1_cmds_pubber"
        pkg="sensors_simulator"
        type="agent1_controls_sim"
        respawn="true" >
        <!-- output="screen" > -->
  </node>

</launch>
